src/modules/auth/service/auth.service.ts

```diff
@@ -39,6 +39,13 @@ class AuthService implements IHttpService {
       data,
     );
   }
+
+  async activateAccount(token: string): Promise<void> {
+    return this.httpClient.postData(
+      `auth/activate-account/${token}`,
+      {},
```

Comment:
W poprzednim PR  chyba było to samo, że body przekazywałeś jako pusty obiekt, nikomu to za bardzo nie wadzi, ale czy nie lepiej zrobić ten parametr opcjonalny? Wtedy jeżeli nie mamy nic do przekazania, to nie przekazujemy - a tak to trzeba pamiętać lub zerkać o co TS krzyczy - aaa okej, nie ma payloadu, to puszczę pusty obiekt.